Makefile

```
@@ -26,18 +26,24 @@ $(BUILDDIR)/android/tun2socks.aar: $(GOMOBILE)
 	mkdir -p "$(BUILDDIR)/android"
 	$(ANDROID_BUILD_CMD) -o "$@" $(IMPORT_PATH)/outline/tun2socks $(IMPORT_PATH)/outline/shadowsocks
 
-apple: $(BUILDDIR)/apple/Tun2socks.xcframework
+apple_legacy: $(BUILDDIR)/apple_legacy/Tun2socks.xcframework
 
-$(BUILDDIR)/apple/Tun2socks.xcframework: $(GOMOBILE)
+$(BUILDDIR)/apple_legacy/Tun2socks.xcframework: $(GOMOBILE)
   # MACOSX_DEPLOYMENT_TARGET and -iosversion should match what outline-client supports.
   # TODO(fortuna): -s strips symbols and is obsolete. Why are we using it?
 	export MACOSX_DEPLOYMENT_TARGET=10.14; $(GOBIND) -iosversion=11.0 -target=ios,iossimulator,macos -o $@ -ldflags '-s -w' -bundleid org.outline.tun2socks $(IMPORT_PATH)/outline/tun2socks $(IMPORT_PATH)/outline/shadowsocks
-	
+
 apple_future: $(BUILDDIR)/apple_future/Tun2socks.xcframework
 
 $(BUILDDIR)/apple_future/Tun2socks.xcframework: $(GOMOBILE)
```

```suggestion
$(BUILDDIR)/macos/Tun2socks.xcframework: $(GOMOBILE)
```

If you move the macos target here.